docs/docs/concepts/redirects.mdx

```diff
@@ -1,9 +1,143 @@
+home screen after a logout or to a specific url on a sub-domain after a settings
+password update. On project creation the redirect flows are all set to the
+default Ory Cloud Managed UI. For more information on the Ory Managed UI, please
+see the [Managed User Interface](./managed-ui.mdx) documentation.
+
+### Login, Registration and Settings
+
+The Login and Registration flows have three fields (all optional):
+
+1. `Default Redirect Url` domain or path
+2. An `Post-Password Redirect Url`
+3. An `Post-OIDC Redirect Url` (**currently not supported**)
+
+And Settings has three fields (all optional):
+
+1. `Default Redirect Url` domain or path
+2. An `Post-Password Redirect Url`
+3. An `Post-Profile Redirect Url`
+
+:::info
+
+Setting `Post-OIDC` redirects for Login and Registration are currently not
+supported since an Ory Cloud project cannot be setup to use OIDC yet. This will
+be supported in the future. For more information see the
+[Ory Cloud Roadmap](../early-access.mdx).
+
+:::
+
+When setting the `Default Redirect Url` as a FQDN (fully qualified domain name)
+such as e.g. https://domain.example/login it will overwrite the
+`Global Redirect Url`. As a path it will replace the `Global Redirect Url` path,
+e.g. a path such as `/login` will alter https://somedomain.example/global/path
+to https://somedomain.example/login.
+
+On the sub-conditions such as `Post-Password` or `Post-OIDC` a FQDN or a path
+can be set. In the case of a FQDN it will replace the closest relative starting
+with the Login/Registration `Default Redirect Url` and then the
+`Global Redirect Url`. With a path it will use the closest relative as the base
+url as described above with the `Default Redirect Url`.
+
+![Browser Redirects Url Management](../images/browser-redirects/browser-redirects-url-management.png)
+
+All the relative urls will be updated with their respective base url. As shown
+in the example below, the Login `Default Redirect Url` has been set to use a
+relative url and thus the base url will be the `Global Redirect Url`
+(https://myapp.com). This will then set the Login `Default Redirect Url` to
+https://myapp.com/after/login and the `Post-Password` to
+https://myapp.com/after/login/password.
+
+![Browser Redirects Url Management Transformed](../images/browser-redirects/browser-redirects-url-management-2.png)
+
+### Verification, Recovery and Logout
+
+:::warning
+
+The `Post-Recovery` redirect is currently not supported and will automatically
+redirect the user to the `Settings UI` url. This implies that you should
+currently rely on the `Post-Settings` redirect url for `Post-Recovery` flows as
+well.
+
+:::
+
+Each of these flows have only a single field, the `Default Redirect Url`. As
+with the Login, Registration and Settings configurations, the
+`Default Redirect Url` can be a FQDN or a path.
+
+![Browser Redirects Url Management Transformed](../images/browser-redirects/browser-redirects-url-management-3.png)
+
+## Resetting/Updating Flows
+
```

Comment:
I would describe here also what happens to existing flows after an update (when someone use same flow ID) and I would mention that changes are immediate meaning new flows will have reflected changes once the configuration update will be populated.